src/main/scala/scalafix/internal/sbt/ScalafixInterfacesClassloader.scala

```diff
@@ -1,19 +0,0 @@
-package scalafix.internal.sbt
-
-/**
-  * A classloader that shares only scalafix-interfaces classes from the parent classloader.
-  *
-  * This classloader is intended to be used as a parent when class-loading scalafix-cli.
-  * By using this classloader as a parent, it's possible to cast runtime instances from
-  * the scalafix-cli classloader into `scalafix.interfaces.Scalafix` from this classlaoder.
-  */
-class ScalafixInterfacesClassloader(parent: ClassLoader)
```

Comment:
Nice! Good riddance

Reply:
Well, it just moved https://github.com/scalacenter/scalafix/blob/v0.9.17/scalafix-interfaces/src/main/java/scalafix/internal/interfaces/ScalafixInterfacesClassloader.java :)

Reply:
Right! It's still nice to have it out of sbt-scalafix so this class doesn't need to be copy-pasted for every client 😊